### DISCUSSION TITLE
Introducing Phoenix Code Desktop- Now on Windows, Mac & Linux

### DISCUSSION THREAD

**Discussion by abose** [phcode-dev/phoenix#1047] [Announcements]
We're thrilled to share a huge update with you all today - Phoenix Code native apps are officially here! 🎉 . Get your copy from https://phcode.io/ 

Discussion: https://www.reddit.com/r/brackets/comments/1b7bihm/introducing_phoenix_code_desktop_now_on_windows/

iOS/Android native apps: https://github.com/phcode-dev/phoenix/discussions/1046

**Comment by Mat0wa3** [comment]
it would be great to have a native windows, linux and mac apps, for me it would be the best, i usually travel a lot and prefer to code while traveling, offline, but phoenix need wifi to work, i want to have all the new features everywhere and anywhere.

**Reply by abose** [reply]
Native builds are top priority right now. Native builds (Beta) for Windows/Mac/Linux should be available by February.

**Comment by denny466** [comment]
I got one: Flathub

Big ❤️ for this webapp, by the way. Messing with HTML and CSS is fun again thanks to your hard work. Best regards!

**Comment by abose** [comment]
**🚀 Exciting News: Introducing Our First Linux Beta Build! 🚀**

Dear Phoenix Code Community,

We are thrilled to announce a major milestone that many of you have eagerly anticipated – the launch of our first Linux beta builds! This achievement marks a significant leap forward in our journey, and it's all thanks to your unwavering support and patience.

**🔧 Installation Instructions**
To dive into the Linux beta version, simply execute the following command in your terminal:
```bash
wget -qO- https://updates.phcode.io/linux/installer.sh | bash
```

**🗑️ Uninstallation**
If you need to uninstall the beta version, please use the following command:
```bash
wget -qO- https://updates.phcode.io/linux/installer.sh | bash -s -- --uninstall
```

**💬 We Want to Hear From You!**
Your feedback is the cornerstone of our improvement. If you encounter any issues or if you have suggestions to make, please share your thoughts with us through our [GitHub issues page](https://github.com/phcode-dev/phoenix/issues/new/choose).

As we continue to refine and enhance Phoenix Code for Linux, we look forward to your invaluable input to make it even better.

**📅 Update: February 11, 2024**

**Comment by abose** [comment]
We're thrilled to share a huge update with you all today - Phoenix Code native apps are officially here! 🎉 . Get your copy from https://phcode.io/ 

Discussion: https://www.reddit.com/r/brackets/comments/1b7bihm/introducing_phoenix_code_desktop_now_on_windows/

iOS/Android native apps: https://github.com/phcode-dev/phoenix/discussions/1046

**Reply by PunyTRexArms** [reply]
Many thanks to all the devs for supporting a Linux version!

**Comment by whitemagehealing** [comment]
Is there a portable version?

**Reply by abose** [reply]
* For mac, the app is already portable, you can just download it from the latest release in https://github.com/phcode-dev/phoenix-desktop/releases and use it.
* For Linux too, the app is portable. Download the zipped files that ends with `amd64_linux_bin-GLIBC-2.33.tar.gz`(for linux distros newer than ubuntu 22), and extract and follow the readme.
* For windows, we dont directly distribute a portable version as no one has asked one yet, but you should be able to install the app and then copy the full folder at "C:\Users\<your user name>\AppData\Local\Phoenix Code\" and zip it. That should be a portable version. 

I have created this discission to track: https://github.com/orgs/phcode-dev/discussions/1820

**Comment by Kolinzoviola** [comment]
That will be great I am waiting for it
